### PR TITLE
components workflow filepath replaced

### PR DESCRIPTION
**Pull request by corinnebosley**

### CODE REVIEW

**Comment by corinnebosley** [comment]
@marqh

File: doc/write_your_own/components/Workflow1.ipynb [hunk 1]
```diff
@@ -405,7 +408,20 @@
...
+    "896b838d96d24f5fbf1fb99e2d9d5c0b": {
+     "views": []
+    },
+    "945eec722a2c47b183dde753cb49d909": {
+     "views": []
+    },
+    "dd3756b52eaa4a16bd8b19e748ca50aa": {
+     "views": []
+    }
+   },
```
Comment (bjlittle) [inline doc/write_your_own/components/Workflow1.ipynb]: @corinnebosley You seem to have some state here ... can this be removed?

Comment (corinnebosley) [inline doc/write_your_own/components/Workflow1.ipynb]: Done everything I can to remove it.  You want me to just delete it by hand? Seems risky...

**Comment by corinnebosley** [comment]
@bjlittle Did it anyway.  That's just the kind of girl I am.  I live on the edge.

**Comment by bjlittle** [comment]
:+1: Thanks!

**Pull request merged**